db/re/item_combo_db.txt

```diff
@@ -490,7 +532,7 @@
 22124:28467:28468,{ bonus bAspdRate,8; bonus bHit,8; bonus2 bExpAddClass,Class_All,2; }
 22125:28469:28470,{ bonus bAspdRate,10; bonus bHit,10; bonus2 bExpAddClass,Class_All,2; }
 22126:28471:28472,{ bonus bAspdRate,10; bonus bHit,10; bonus2 bExpAddClass,Class_All,2; bonus bSpeedAddRate,25; }
-24012:24013:24014:24015:24016:24017,{ bonus bAllStats,9; }
+24012:24013:24014:24015:24016:24017,{ bonus bAllStats,3; }
```

Comment:
It's in the commit message here
https://github.com/rathena/rathena/pull/1848/commits/a4251f61d1e41d9bcc2927160e00b8b0a97b77fd